src/includes/class-sign-in-with-google.php

```diff
@@ -181,6 +181,7 @@ private function define_admin_hooks() {
 		$this->loader->add_action( 'admin_init', $plugin_admin, 'process_settings_import' );
 		$this->loader->add_action( 'admin_enqueue_scripts', $plugin_admin, 'enqueue_styles' );
 		$this->loader->add_action( 'show_user_profile', $plugin_admin, 'add_connect_button_to_profile' );
+		$this->loader->add_action( 'login_init', $plugin_admin, 'check_login_redirection', 888 );
```

Comment:
I wonder if there is a better place to check for the login form? Setting the priority to 888 seems like an arbitrary number and I want to make sure we're doing things the best we can.

Reply:
Sure, I don't say 888 is good priority - I am sure you can set that number better.  also, there might be also better places (compared to `login_init`) but at this moment I am not in my workspace to check which one is better. though , iirc, `login_init` was suitable. the priority (number) - you can set to `10` too, or any you want.